src/test/intmath_tests.cpp

```diff
@@ -1,61 +1,146 @@
-#include <util/intmath.h>
-
-#include <cmath>
+// Copyright (c) 2021 The Bitcoin developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #include <boost/test/unit_test.hpp>
+#include <test/lcg.h>
 #include <test/util/setup_common.h>
 
+#include <script/intmath.h>
+
+#include <boost/multiprecision/cpp_int.hpp>
+
 BOOST_FIXTURE_TEST_SUITE(intmath_tests, BasicTestingSetup)
 
-BOOST_AUTO_TEST_CASE(log2fixedpoint_tests) {
```

Comment:
Where did this go? I can't find it. Best is probably to merge both files

Reply:
Yeah, oops.